<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="TableWidget" name="tablePage"/>
      <widget class="BallsWidget" name="ballsPage"/>
      <widget class="HitWidget" name="hitPage"/>
      <widget class="ViewWidget" name="viewPage"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TableWidget</class>
   <extends>QWidget</extends>
   <header location="global">gui/tablewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>HitWidget</class>
   <extends>QWidget</extends>
   <header location="global">gui/hitwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ViewWidget</class>
   <extends>QWidget</extends>
   <header location="global">gui/viewwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BallsWidget</class>
   <extends>QWidget</extends>
   <header location="global">gui/ballswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
